public/app/features/live/live.ts

```diff
@@ -74,55 +59,6 @@ export class GrafanaLiveService implements GrafanaLiveSrv {
    * Join and leave messages will be sent to the open stream
    */
   async getPresence(address: LiveChannelAddress): Promise<LiveChannelPresenceStatus> {
-    const channelConfig = await this.getChannelInfo(address);
-    return this.deps.centrifugeSrv.getPresence(address, channelConfig);
-  }
-
-  /**
-   * Get a channel.  If the scope, namespace, or path is invalid, a shutdown
-   * channel will be returned with an error state indicated in its status.
-   *
-   * This is a singleton instance that stays active until explicitly shutdown.
-   * Multiple requests for this channel will return the same object until
-   * the channel is shutdown
-   */
-  async getChannelInfo(addr: LiveChannelAddress): Promise<LiveChannelConfig> {
-    if (!isValidLiveChannelAddress(addr)) {
```

Comment:
The `isValidLiveChannelAddress` check will be missing from `getDataStream` after this change. Not sure if this is important, as I assume we can trust the value passed to this function

Reply:
Ultimately the backend will give an error -- this was a check to validate before sending.  We likely should keep it somewhere, perhaps before calling subscribe

Reply:
added the check and fast failure to:
https://github.com/grafana/grafana/pull/41584/files#diff-3c0c5be9d18600a8b127a2ef276d708f289a68140ee066eb389ce984aa99ddc5R50